### ISSUE TITLE
Screenreaders in Content Need to be Two Words

### ISSUE THREAD

**Issue by jfc3-dol** [mdn/content#19738]
### MDN URL

https://developer.mozilla.org/en-US/docs/Learn/Accessibility/WAI-ARIA_basics

### What specific section or headline is this issue about?

Dynamic content updates

### What information was incorrect, unhelpful, or incomplete?

In the following text "...the content update is not detected by **screenreaders**..." the words screen readers should be two words instead of one. Made the text I'm referencing bold to make it easier to spot.

Please check the rest of the page for other occurrences where screenreader is one word and correct them.

**NOTE** - I searched "screenreader", and there were 47 cases of if being one word instead of two. You might want to do further searches to fix any other occurrences of the problem on other pages.

### What did you expect to see?

I expected to see "....the content update is not detected by **screen readers**...". Made the text I'm referencing bold to make it easier to spot.

### Do you have any supporting links, references, or citations?

nope

### Do you have anything more you want to share?

nope
